### PR TITLE
Update / Add links to full lists of related data

### PR DESCRIPTION
**Pull request by DurtyFree**
This updates and adds several links to full lists of related data, links point to this GitHub repo: https://github.com/DurtyFree/gta-v-data-dumps which is maintained and updated by me for each GTA V update

### CODE REVIEW

File: natives.json [hunk 1]
```diff
@@ -2379,7 +2379,7 @@
 		},
 		"0x651D3228960D08AF": {
 			"name": "_LINK_STATIC_EMITTER_TO_ENTITY",
-			"jhash": "",
+			"jhash": "Full list of static emitters by DurtyFree: https://github.com/DurtyFree/gta-v-data-dumps/blob/master/staticEmitters.json",
```
Comment (alloc8or) [inline natives.json:2382]: Wrong field.

File: natives.json [hunk 2]
```diff
@@ -24195,7 +24195,7 @@
 		},
 		"0xE35B38A27E8E7179": {
 			"name": "_ANIMPOSTFX_GET_UNK",
-			"jhash": "",
+			"jhash": "Full list of animpostFX / screen effects by DurtyFree: https://github.com/DurtyFree/gta-v-data-dumps/blob/master/animPostFxNamesCompact.json",
```
Comment (alloc8or) [inline natives.json:24198]: Wrong field.

**Review by alloc8or — CHANGES_REQUESTED** [review]

File: natives.json (hunk 2 above)
Comment (DurtyFree) [reply natives.json:24198]: Fixed thanks

File: natives.json (hunk 1 above)
Comment (DurtyFree) [reply natives.json:2382]: Fixed thanks

**Comment by DurtyFree** [comment]
I have fixed the comments at the wrong fields

**Pull request merged**